types/google-ads-scripts/index.d.ts

```diff
@@ -0,0 +1,158 @@
+// Type definitions for Google Ads Scripts 1.0
+// Project: https://developers.google.com/google-ads/scripts
+// Definitions by: JJPell <https://github.com/JJPell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
```

Comment:
```suggestion
// Minimum TypeScript Version: 4.1
```
why this high version? Your opting to ES5 in your setup (including older syntax in tests), while requiring TS4.1 runtime features for package consumption

Reply:
can you try different approaches (you can skip this, if that is not feasable), instead of using template literals types? so like:
```ts
 keyof typeof ExtensionScheduleDayOfWeek;
 ```

Reply:
so like:
```ts
const ExtensionScheduleDayOfWeek: {
            Monday: "MONDAY",
            Tuesday: "TUESDAY",
            Wednesday: "WEDNESDAY",
            Thursday: "THURSDAY",
            Friday: "FRIDAY",
            Saturday: "SATURDAY",
            Sunday: "SUNDAY",
        };

        type ExtensionScheduleDayOfWeekType =(typeof ExtensionScheduleDayOfWeek)[keyof typeof ExtensionScheduleDayOfWeek];
```

Reply:
I knon, klunky, but why do you need those literals?

Reply:
The reason the version was 4.1 was because I was making use of the new String enums feature which allows you to convert them to a String literal type by doing the following:

```typescript
        /** Enum of supported extension sechedule days of the week */
        enum ExtensionScheduleDayOfWeek {
            Monday = "MONDAY",
            Tuesday = "TUESDAY",
            Wednesday = "WEDNESDAY",
            Thursday = "THURSDAY",
            Friday = "FRIDAY",
            Saturday = "SATURDAY",
            Sunday = "SUNDAY",
        }

        type ExtensionScheduleDayOfWeekType = `${ExtensionScheduleDayOfWeek}`;
```

In hindsight I agree. I think its trying to be too elegant using enums instead of just defining a literal, at the expense of the typescript version. The enums are there as I wanted the user to be able to import and use them as needed, rather than just defining String literal types.

I'll convert the enums to literals and update you once ready.

Reply:
Thanks, really appreciated! Yes, I'd pick wider usage and frictionless interop with other definitions!

Reply:
No worries! I've made the change now 👍

Reply:
Thanks!
My intent was to point, that when consuming those types, it should be easy to use different code
paradigms, without using `enum`. Same line below is just as possible with enum and with consts:
```ts
const day: Day = Days.Monday || "sunday";
```